TUM Campus App/TumSexyManager.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TumSexyManager.swift
+//  TUM Campus App
+//
+//  Created by Mathias Quintero on 3/28/17.
+//  Copyright © 2017 LS1 TUM. All rights reserved.
+//
+
+import Alamofire
+import SwiftyJSON
+import Sweeft
+
+class TumSexyManager: Manager {
+    
+    required init(mainManager: TumDataManager) {
+        // Nothing required
+    }
+    
+    func fetchData(_ handler: @escaping ([DataElement]) -> ()) {
+        Alamofire.request("http://json.tum.sexy").responseJSON { response in
```

Comment:
Well, it's probably a bit too strict calling this a critical/blocking issue, my friend @TCA-Bot